src/decode.jl

```diff
@@ -0,0 +1,40 @@
+using ColorTypes
+using ColorVectorSpace
+using FixedPointNumbers
+using FileIO
+
+function gif_decode(path)
+    Error = Cint(0)
+    gif = LibGif.DGifOpenFileName(path, Ref(Error))
```

Comment:
If this opens a file handler, then it should be properly closed inside this function. Use

```julia
try
    gif = LibGif.DGifOpenFileName(path, Ref(Error))
    ...
finally
    LibGif.EGifCloseFile(gif, Ref(Error));
end
```

This is for the same reason to https://github.com/ashwani-rathee/ExifViewer.jl/pull/3#discussion_r913669820